crates/detect-targets/src/detect/linux.rs

```diff
@@ -31,32 +32,24 @@ pub(super) async fn detect_targets(target: String) -> Vec<String> {
         //
         // As such, we need to launch the test ourselves.
         Libc::Gnu | Libc::Musl => {
-            let cpu_arch = target
+            let (cpu_arch, _) = target
                 .split_once('-')
-                .expect("unwrap: target always has a - for cpu_arch")
-                .0;
+                .expect("unwrap: target always has a - for cpu_arch");
 
             let cpu_arch_suffix = cpu_arch.replace('_', "-");
 
-            let handles: Vec<_> = [
-                format!("/lib/ld-linux-{cpu_arch_suffix}.so.2"),
-                format!("/lib/{cpu_arch}-linux-gnu/ld-linux-{cpu_arch_suffix}.so.2"),
-                format!("/usr/lib/{cpu_arch}-linux-gnu/ld-linux-{cpu_arch_suffix}.so.2"),
-            ]
+            let has_glibc = futures_util::future::join_all(
```

Comment:
Please use tokio::spawn as before, its performance is better than joining due to multithreading.

Since vfork + exec will block the calling thread, using spawn makes more sense here.